config/routes.rb

```diff
@@ -44,6 +44,10 @@
   ActiveAdmin.routes(self)
 
   namespace :api, path: '/api/v1', defaults: {format: :json} do
+    namespace :popolo do
```

Comment:
My gripe with moving this out of the api namespace at the moment is that the RestfulController is in the API namespace, so otherwise we get something like

```
class Popolo::MotionsController < API::RestfulController # hrm..
```

And a folder structure like:
```
- popolo
  - motions_controller # inherits from a different folder?
- api
  - restful_controller
```